x/ecocredit/basket/msg_create.go

```diff
@@ -12,39 +15,38 @@ var (
 	_ legacytx.LegacyMsg = &MsgCreate{}
 )
 
-const nameMaxLen = 32
-const displayNameMinLen = 3
-const displayNameMaxLen = 32
+const nameMinLen = 3
+const nameMaxLen = 8
+const descrMaxLen = 200
 const exponentMax = 32
-const creditNameMaxLen = 32
+const creditAbbrMaxLen = 32
```

Comment:
This should be maxLen of 3 now that we're using credit type abbreviation instead of credit type name: https://github.com/regen-network/regen-ledger/blob/ba2f1a28b3270e04a164ec2dee456813e18db7a6/x/ecocredit/params.go#L151-L158

Reply:
hmm, we should expose that and use common validation. I will update the code to reuse this function.